File: LinguagemC/IF/imc.c
#include<stdio.h>

int main() {
float peso, altura, imc;


printf("----------------IMC----------------\n");
printf("Digite o peso (kg): ");
scanf("%f", &peso);

printf("Digite a altura (m): ");
scanf("%f", &altura);
 
imc = peso / (altura * altura);
printf("Seu IMC é: %.2f\n", imc);

printf("-----------------------------------\n");

if (imc < 18.5){ 
printf("Classificação: Abaixo do peso\n");

}else if (imc >= 18.5 && imc < 24.9){
printf("Classificação: Peso normal\n");

}else if (imc >= 25 && imc < 29.9){
printf("Classificação: Sobrepeso\n");

}else if (imc >= 30 && imc < 34.9){
printf("Classificação: Obesidade grau 1\n");

}else if (imc >= 35 && imc < 39.9){
printf("Classificação: Obesidade grau 2\n");

}else{
 printf("Classificação: Obesidade grau 3 (mórbida)\n");
}
 
return 0;
}

File: Python/excecoes/imc.py
try:
    peso = float(input("Digite seu peso em kg: "))
    altura = float(input("Digite sua altura em metros: "))
    
    imc = peso / (altura ** 2)
    print(f"Seu IMC é: {imc:.2f}")
    
    if imc < 18.5:
        print("Você está abaixo do peso.")
    elif imc < 24.9:
        print("Você está com o peso normal.")
    elif imc < 29.9:
        print("Você está com sobrepeso.")
    elif imc < 34.9:
        print("Você está com obesidade grau I.")
    elif imc < 39.9:
        print("Você está com obesidade grau II.")
    else:
        print("Você está com obesidade grau III.")
        
except ValueError:
    print("Erro: insira apenas valores numéricos válidos.")